package.json

```diff
@@ -74,27 +75,26 @@
     "eslint-plugin-jsdoc": "^37.9.7",
     "eslint-plugin-jsx-a11y": "^6.6.1",
     "eslint-plugin-prettier": "^4.2.1",
-    "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-react": "^7.31.10",
+    "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-tsdoc": "^0.2.17",
-    "eslint": "^8.26.0",
     "ethers": "^5.7.2",
     "glob": "^10.2.6",
     "husky": "^8.0.3",
-    "jest-text-transformer": "^1.0.4",
     "jest": "^29.5.0",
+    "jest-text-transformer": "^1.0.4",
     "nodemon": "^2.0.22",
     "npm-run-all": "^4.1.5",
     "open": "^8.4.0",
     "prettier": "^2.8.8",
     "shelljs": "^0.8.5",
-    "textlint-rule-no-dead-link": "^5.1.2",
     "textlint": "^13.3.2",
+    "textlint-rule-no-dead-link": "^5.1.2",
     "ts-generator": "^0.1.1",
     "ts-jest": "^29.0.3",
     "tsup": "^6.7.0",
     "tsx": "^3.12.7",
     "turbo": "^1.8.8",
-    "typescript": "^4.8.4"
+    "typescript": "^5.1.6"
```

Comment:
Typescript doesn't follow semver; each of their minor versions may contain a breaking change. Because of that, we should write ~5.1.6 instead of ^5.1.6.

Reply:
Good catch!